VogtPayroll5/PayrollConsoleReader.cs

```diff
@@ -24,5 +24,31 @@ public Employee GetEmployeeInfo()
 
             return emp;
         }
+
+        public bool ReturnTrueOrFalse()
+        {
+            string temp;
+            Console.WriteLine("Would you like to add another employee? y/n");
+            temp = Console.ReadLine().ToLower();
+
+            do
+            {
+                if (temp == "y")
+                {
+                    return true;
+                }
+                else if (temp == "n")
+                {
+                    return false;
+                }
+                else
+                {
+                    Console.WriteLine("Please enter 'y' or 'n'!");
+                    temp = Console.ReadLine().ToLower();
+                }
+            } while (temp != "y" || temp == "n");
+
+            return true;
```

Comment:
Do you really want the default to be true? Typically, the default for true or false is false, but it always depends on your context. Reading from the console seems generic enough that you might want to default to false.

Could always make it optional.

```c#
public bool ReadTrueOrFalse(bool defaultValue = false) {
.
.
.
.
.
return defaultValue
}
```

Then, you can call it either of two ways:
```c#
// this will use the default behavior of "false"
ReadTrueOrFalse()

// this will force the default behavior to "true"
ReadTrueOrFalse(true)
```